src/test/java/gameboy/gamepack/racinggame/data/vo/PositionTest.java

```diff
@@ -0,0 +1,33 @@
+package gameboy.gamepack.racinggame.data.vo;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+class PositionTest {
```

Comment:
클래스에 DisplayName을 선언하면 테스트를 돌렸을 때
클래스 이름 대신 정의한 이름이 출력됩니다. 이러면 여러 테스트 케이스를 돌렸을 때 좀 더 쉽게 파악할 수 있어요.

- 예시
```suggestion

@DisplayName("도메인 - Position 테스트")
class PositionTest {
```
개인적인 의견이니까 참고만 해주세요